content/id/docs/concepts/services-networking/connect-applications-service.md

```diff
@@ -0,0 +1,366 @@
+
+Kamu dapat membuat *Service* untuk replika 2 *nginx* dengan `kubectl explose`:
+
+```shell
+kubectl expose deployment/my-nginx
+```
+```
+service/my-nginx exposed
+```
+
+Perintah diatas sama dengan `kubectl apply -f` dengan *yaml* sebagai berikut:
+
+{{< codenew file="service/networking/nginx-svc.yaml" >}}
+
+Spesifikasi ini akan membuat *Service* yang membuka *TCP port 80* disetiap *Pod* dengan label `run: my-nginx` dan mengeksposnya kedalam *port Service* (`targetPort`: adalah port kontainer yang menerima trafik, `port` adalah *service port* yang dapat berupa *port* apapun yang digunakan *Pod* lain untuk mengakses *Service*).
+
+Lihat [Service](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#service-v1-core)
+API object untuk melihat daftar *field* apa saja yang didukung di *service definition*. Cek *Service* kamu:
+
+```shell
+kubectl get svc my-nginx
+```
+```
+NAME       TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)   AGE
+my-nginx   ClusterIP   10.0.162.149   <none>        80/TCP    21s
+```
+
+Seperti yang disebutkan sebelumnya, sebuah *Service* berisi grup *Pod*. *Pod* diekspos melalui `endpoints`. *Service selector* akan mengecek *Pod* secara terus-menerus dan hasilkan akan dikirim (*POSTed*) ke objek *endpoint* yang bernama `my-nginx`. Saat sebuah *Pod* mati, *IP Pod* didalam *endpoint* tersebut akan otomatis dihapus, dan *Pod* baru yang sesuai dengan *Service selector* akan otomatis ditambahkan kedalam *endpoint*. Cek *endpoint* dan perhatikan bahwa *IP* sama dengan *Pod* yang dibuat di langkah pertama: 
+
+```shell
+kubectl describe svc my-nginx
+```
+```
+Name:                my-nginx
+Namespace:           default
+Labels:              run=my-nginx
+Annotations:         <none>
+Selector:            run=my-nginx
+Type:                ClusterIP
+IP:                  10.0.162.149
+Port:                <unset> 80/TCP
+Endpoints:           10.244.2.5:80,10.244.3.4:80
+Session Affinity:    None
+Events:              <none>
+```
+```shell
+kubectl get ep my-nginx
+```
+```
+NAME       ENDPOINTS                     AGE
+my-nginx   10.244.2.5:80,10.244.3.4:80   1m
+```
+
+Kamu sekarang dapat melakukan *curl* kedalam *nginx Service* di `<CLUSTER-IP>:<PORT>` dari *node* manapun di kluster. Perlu dicatat bahwa *Service IP* adalah *IP virtual*, *IP* tersebut tidak pernah ada di *interface node* manapun. Jika kamu penasaran bagaimana konsep ini bekerja, kamu dapat membaca lebih lanjut tentang [service proxy](/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies).
+
+## Mengakses Service
+
+Kubernetes mendukung 2 mode utama untuk menemukan sebuah *Service* - variabel *Environment* dan *DNS*. 
```

Comment:
I remember that we agree on using variabel `environment` based on slack discussion

Reply:
so `environment` without capital `E`?